src/Entity/Theme.php

```diff
@@ -13,22 +13,59 @@ class Theme
     #[ORM\Column]
     private ?int $id = null;
 
```

Comment:
Don't do this. Leave the name alone. Let doctrine do the standard transform. Standard is camel in PHP. Snake in db